xml/System.Net.Sockets/SocketTaskExtensions.xml

```diff
@@ -634,12 +634,12 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="3" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="socket">To be added.</param>
-        <param name="buffer">To be added.</param>
-        <param name="socketFlags">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="socket">The socket to perform the operation on.</param>
+        <param name="buffers">A region of memory that contains the data to send.</param>
+        <param name="socketFlags">A bitwise combination of the <see cref="T:System.Net.Sockets.SocketFlags" /> values.</param>
+        <param name="cancellationToken">A cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
+        <summary>Sends data to a connected socket.</summary>
+        <returns>An asynchronous task that completes with number of bytes sent to the socket if the operation was successful. Otherwise, the task will complete with an invalid socket error.</returns>
```

Comment:
Not all tasks are asynchronous. Sends can often write immediately to the OS buffer.
```suggestion
        <returns>A task that completes with number of bytes sent to the socket if the operation was successful. Otherwise, the task will complete with an invalid socket error.</returns>
```